src/Compilers/Core/Portable/Operations/ControlFlowGraphBuilder.cs

```diff
@@ -1940,37 +1992,25 @@ public override IOperation VisitInvocation(IInvocationOperation operation, int?
                 _evalStack.Push(Visit(operation.Instance));
             }
 
-            ImmutableArray<IArgumentOperation> arguments = operation.Arguments;
-            foreach (IArgumentOperation argument in arguments)
-            {
-                _evalStack.Push(Visit(argument.Value));
-            }
-
-            ImmutableArray<IArgumentOperation> visitedArguments;
-            if (!arguments.IsEmpty)
-            {
-                var builder = ArrayBuilder<IArgumentOperation>.GetInstance();
-                for (int i = arguments.Length - 1; i >= 0; i--)
-                {
-                    IOperation visitedValue = _evalStack.Pop();
-                    var originalArgument = (BaseArgument)arguments[i];
-                    builder.Add(new ArgumentOperation(visitedValue, originalArgument.ArgumentKind, originalArgument.Parameter,
-                                                      originalArgument.InConversionConvertibleOpt, originalArgument.OutConversionConvertibleOpt,
-                                                      semanticModel: null, originalArgument.Syntax, originalArgument.ConstantValue, originalArgument.IsImplicit));
-                }
-                builder.ReverseContents();
-                visitedArguments = builder.ToImmutableAndFree();
-            }
-            else
-            {
-                visitedArguments = ImmutableArray<IArgumentOperation>.Empty;
-            }
-
+            ImmutableArray<IArgumentOperation> visitedArguments = VisitArguments(operation.Arguments);
             IOperation visitedInstance = operation.Instance == null ? null : _evalStack.Pop();
 
             return new InvocationExpression(operation.TargetMethod, visitedInstance, operation.IsVirtual, visitedArguments, semanticModel: null, operation.Syntax, operation.Type, operation.ConstantValue, operation.IsImplicit);
         }
 
+        public override IOperation VisitObjectCreation(IObjectCreationOperation operation, int? captureIdForResult)
+        {
+            ImmutableArray<IArgumentOperation> visitedArgs = VisitArguments(operation.Arguments);
+
+            // Initializer is removed from the tree and turned into a series of statements that assign to the created instance
+            var objectCreation = new ObjectCreationExpression(operation.Constructor, initializer: null, visitedArgs, semanticModel: null, operation.Syntax, operation.Type, operation.ConstantValue, operation.IsImplicit);
+
+            // PROTOTYPE(dataflow): For a non-null initializer, we'll need to assign the created object to a flow reference and rewrite all initializers to assign to/call functions on it, then return return the flow reference
```

Comment:
Presuming this is next on your list - otherwise a tracking issue might be helpful.

Reply:
This is the next thing I'm working on.